lib/commits.js

```diff
@@ -33,6 +33,7 @@ module.exports.findCommitsWithAssociatedPullRequestsQuery = /* GraphQL */ `
                   nodes {
                     title
                     number
+                    body
```

Comment:
let me regenerate after adding some body to those PRs :)

Reply:
Perhaps the GraphQL can optionally only query for this if `change-template` includes `$BODY` thus reducing the amount of data required and reducing the time to response?

Reply:
Good call on this when I tested it I noticed the difference in the time to respond.